crates/ra_prof/Cargo.toml

```diff
@@ -1,17 +1,19 @@
-[package]
-edition = "2018"
-name = "ra_prof"
-version = "0.1.0"
-authors = ["rust-analyzer developers"]
-publish = false
-
-[dependencies]
-once_cell = "1.0.1"
-itertools = "0.8.0"
-backtrace = "0.3.28"
-jemallocator = { version = "0.3.2", optional = true }
-jemalloc-ctl = { version = "0.3.2", optional = true }
-
-[features]
-jemalloc = [ "jemallocator", "jemalloc-ctl" ]
-cpu_profiler = []
+[package]

```

Comment:
What happened here? Is git being weird about newlines?

Reply:
Hm... so I'm on windows and I just noticed that globally I have `core.autocrlf=true` but locally it's false... that seems wrong.

I think the correct fix is specifying everything as LF in .gitattributes?

Reply:
No idea. A while ago I thought I understood how these worked, but then I realized I didn't.